src/isar/storage/uploader.py

```diff
@@ -85,6 +85,9 @@ def run(self) -> None:
 
                 inspection, mission = self.upload_queue.get(timeout=1)
 
+                if mission == None:
+                    raise Exception("Mission cannot be null")
```

Comment:
This exception will not be caught anywhere as far as I can see which would lead to the Uploader thread crashing and thus the rest of the application crashing. 

As we are doing communication internally within the application it might not be neccessary with this None check at all and I would suggest removing it.

Reply:
If the mission is None it will crash if we raise and Exception or not, so I guess we change from raising Exception to not trying to upload?

Reply:
Is it not negative to use a continue there without notifying further upstream? As the user in flotilla for example might think the upload has been performed.

Reply:
We need to log something yes, Im thinking perhaps log an error or a warning

Reply:
My point is that this is internal communication for the application and if designed properly the mission should never be `None`. It was `None` now due to a bug, not because of some input that was wrong. My suggestion then is to remove the check for `null` entirely because a failure here implies that we have a bug somewhere else, not that something happened which might happen once in awhile.

Reply:
If you decide that the input should be checked for `None` then make sure it's also caught in the `Uploader` thread such that it won't crash. In that case you should of course log something.